src/+otp/+constrainedpendulum/+presets/Canonical.m

```diff
@@ -0,0 +1,22 @@
+classdef Canonical < otp.constrainedpendulum.ConstrainedPendulumProblem
+    %CANONICAL The constrained pendulum problem
+    %
+    % See
+    %    Ascher, Uri. "On symmetric schemes and differential-algebraic equations."
+    %    SIAM journal on scientific and statistical computing 10.5 (1989): 937-949.
```

Comment:
Is this the right source?  I don't see anything on a pendulum.

Reply:
Hairer, E., Roche, M., Lubich, C. (1989). Description of differential-algebraic problems. In: The Numerical Solution of Differential-Algebraic Systems by Runge-Kutta Methods. Lecture Notes in Mathematics, vol 1409. Springer, Berlin, Heidelberg. https://doi.org/10.1007/BFb0093948

That is the pendulum problem source. 

The Ascher book has a nice formulation we use.

Reply:
Yeah the citation is wrong.